src/Microsoft.VisualStudio.Debugger.Interop.DAP/Microsoft.VisualStudio.Debugger.Interop.DAP.cs

```diff
@@ -35,4 +35,21 @@ public interface IDebugExpressionDAP
         [PreserveSig]
         int EvaluateSync([In] enum_EVALFLAGS dwFlags, [In] DAPEvalFlags dapFlags, [In] uint dwTimeout, [In][MarshalAs(UnmanagedType.Interface)] IDebugEventCallback2 pExprCallback, [Out, MarshalAs(UnmanagedType.Interface)] out IDebugProperty2 ppResult);
     }
+
+    /// <summary>
+    /// 
+    /// </summary>
+    [ComImport()]
+    [ComVisible(true)]
+    [Guid("1615C23F-2BA4-4018-A135-E8B889B01F3E")]
+    [InterfaceType(1)]
+    public interface IDebugProgramDAP
+    {
+        /// <summary>
+        /// 
```

Comment:
Add something for the description? And be clear that the pointer size should be in bits (ex: typically `64` or `32`).